mempool/src/lib.rs

```diff
@@ -556,7 +582,8 @@ pub enum ReturnCode {
     FeeTooLow,
     Invalid,
     Accepted,
-    Known
+    Known,
+    Filtered
```

Comment:
For easy extendibility, add a `,`.
```suggestion
    Filtered,
```